packages/data-explorer/src/lib/api.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
+  baseURL: '/',
+  timeout: 1000,
+  validateStatus: function (status) {
+    if (status === 401) {
```

Comment:
what about 404 ?

Reply:
Volgens mij is dat puur een configuratie fout geweest in mijn lokale Molgenis instance. Een 401 is de correcte code. Op basis van een 404 zou je volgens mij niet de aanname mogen maken dat er ingelogd moet worden. Anders moeten we het misschien op een andere manier oplossen.